packages/google_atlas/lib/src/google_atlas.dart

```diff
@@ -56,14 +57,26 @@ class _GoogleMapsProviderState extends State<GoogleMapsProvider> {
   Widget build(BuildContext context) {
     _onPositionUpdate();
 
-    return GoogleMaps.GoogleMap(
-      myLocationEnabled: showMyLocation,
-      myLocationButtonEnabled: showMyLocationButton,
-      mapType: GoogleMaps.MapType.normal,
-      initialCameraPosition: _toGoogleCameraPosition(position),
-      markers: markers.map((m) => _toGoogleMarker(m)).toSet(),
-      onTap: _toGoogleOnTap(onTap),
-      onMapCreated: _onMapCreated,
+    return FutureBuilder(
+      future: _toGoogleMarkers(markers),
+      builder: (
+        BuildContext context,
+        AsyncSnapshot<Set<GoogleMaps.Marker>> snapshot,
+      ) {
+        if (snapshot.data == null) {
+          return Container();
```

Comment:
I don't think we should delay rendering the map just for the markers. We can show the map and then update the markers as we load them.

Reply:
Yep, good catch... that experience should be more pleasant to the user